src/messages/discord_embeds.py

```diff
@@ -1,19 +1,21 @@
 from .generic_api import GenericApi
 from config import Config
+from logger import Logger
 import cv2
 import datetime
 import traceback
 import discord
 from version import __version__
 import numpy as np
-from discord import Webhook, RequestsWebhookAdapter, Color
+from discord import Webhook, RequestsWebhookAdapter, Color, InvalidArgument
 
 class DiscordEmbeds(GenericApi):
     def __init__(self):
         self._config = Config()
-        self._webhook = Webhook.from_url(self._config.general['custom_message_hook'], adapter=RequestsWebhookAdapter(), )
-        self._file = None
-        self._psnURL = "https://i.psnprofiles.com/games/3bffee/trophies/"
```

Comment:
Why removal of these two variables?

Reply:
@egut125 Ah not sure how those lines got deleted. I just pushed a commit to restore them. Good catch.